docs/core/install/includes/linux-install-31-zyp.md

```diff
@@ -15,7 +15,7 @@ The .NET Core Runtime allows you to run apps that were made with .NET Core that
 sudo zypper install aspnetcore-runtime-3.1
 ```
 
-As an alternative to the ASP.NET Core Runtime, you can install the .NET Core Runtime that doesn't include ASP.NET Core support: replace `aspnetcore-runtime-2.1` in the previous command with `dotnet-runtime-3.1`.
+As an alternative to the ASP.NET Core Runtime, you can install the .NET Core Runtime that doesn't include ASP.NET Core support: replace `aspnetcore-runtime-3.1` in the command above with `dotnet-runtime-3.1`.
```

Comment:
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above

Same change should be done in other places in this PR too

```suggestion
As an alternative to the ASP.NET Core Runtime, you can install the .NET Core Runtime that doesn't include ASP.NET Core support: replace `aspnetcore-runtime-3.1` in the previous command with `dotnet-runtime-3.1`.
```

Reply:
We settled a while back on this case to uppercase Runtime. I know it's not the usual.

Reply:
It's not just unusual, it's inconsistent -- I see everywhere in theses same articles "the runtime" and "the runtimes" lowercase. It's not too late to reconsider the capitalization here, especially since everywhere else I've seen the term uses lowercase.